.changeset/nasty-ligers-remember.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/cms-api": minor
+---
+
+Allow setting environment for User Permissions Module
```

Comment:
Where do I get the environment from in the app? NODE_ENV is production for our applications in all deployed envs if I remember correctly

Reply:
I also don't understand how this is planned to be used. The name `environment` is maybe too unspecific.

And I remember we talked about this, but can you please add in the description what problem this solves?